projects/noloan/app/src/main/java/hasadna/noloan/Firestore/FirestoreElement.java

```diff
@@ -1,5 +1,6 @@
-package hasadna.noloan2;
+package hasadna.noloan.Firestore;
```

Comment:
Package names (and corresponding folder names) should be lowercase (also for `Lawsuit`).